### PR TITLE
Update schema.rs for metadata json parsing after changes

### PR DESCRIPTION
**Pull request by d-bhutada**
# Description
The description of the main changes of your pull request

# Related Issue(s)

# Documentation

### CODE REVIEW

File: rust/src/schema.rs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #![allow(non_snake_case, non_camel_case_types)]
 
 use serde::{Deserialize, Serialize};
+use serde_json::Value
```
Comment (langelgjm) [inline rust/src/schema.rs:5]: @d-bhutada this line needs a semicolon at the end.

**Review by langelgjm — CHANGES_REQUESTED** [review]

**Comment by langelgjm** [comment]
This PR is the same as #531 which builds locally for me.

**Comment by langelgjm** [comment]
I can confirm that this resolves #457 and #524 for me.

**Comment by houqp** [comment]
thanks @d-bhutada , closing this since #531 landed first ;)

**Pull request closed without merging**